launcherscript/src/main/java/net/minecraftforge/launcherscript/LauncherScript.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Minecraft Forge - Forge Development LLC
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.launcherscript;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+
+public class LauncherScript
+{
+
+    public static void main(String[] args) throws Exception
+    {
+
+        List<String> command = new ArrayList<>();
+        Collections.addAll(command, findJavaBinary(), "@user_jvm_args.txt");
+
+        command.add(String.format(Locale.ROOT, "@libraries/%s/%s_args.txt", loadMavenPath(), isWindows() ? "win" : "unix"));
+        Collections.addAll(command, args);
+        new ProcessBuilder(command)
+                .directory(null)
+                .inheritIO()
+                .start().waitFor();
```

Comment:
Is there a reason as to why the launcher process isn't shut down?

Reply:
Because that would make whatever server software is running (usually the panel of a hoster or something like Multicraft) think that the server stopped.